src/main/java/org/jetbrains/bsp/bazel/server/util/ParsingUtils.java

```diff
@@ -47,6 +48,47 @@ public static StatusCode parseExitCode(int exitCode) {
     }
   }
 
+  public static String convertOutputToPath(String output, String prefix) {
+    String pathToFile = output.replaceAll("(//|:)", "/");
+    return prefix + pathToFile;
+  }
+
+  public static String getRuleType(String ruleClass) {
+    if (ruleClass.contains(Constants.LIBRARY_RULE_TYPE)) {
```

Comment:
it can be done with map

Reply:
Same as [here](https://github.com/JetBrains/bazel-bsp/pull/24#discussion_r549536722).